src/sentry/static/sentry/app/components/assistant/supportDrawer.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import $ from 'jquery';
+import HookStore from '../../stores/hookStore';
+
+// SupportDrawer slides up when the user clicks on a "Need Help?" cue.
+const SupportDrawer = createReactClass({
+  displayName: 'SupportDrawer',
+
+  propTypes: {
+    closeHandler: PropTypes.func.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      inputVal: '',
+      docResults: [],
+      helpcenterResults: [],
+    };
+  },
+
+  componentWillReceiveProps(props) {
+    this.setState({inputVal: ''});
+  },
+
+  handleSubmit(evt) {
+    evt.preventDefault();
+  },
+
+  handleChange(evt) {
+    evt.preventDefault();
+    let term = encodeURIComponent(evt.currentTarget.value);
```

Comment:
damn, jquery doesn't do this for you?

Reply:
Hmm I copied this from a search handler in getsentry, jquery does handle this.